music-playlist-app/index.js

```diff
@@ -103,6 +121,10 @@ function goToNextSong() {
   currentSong === playlist.length - 1 ? (currentSong = 0) : currentSong++;
   song = new Audio(playlist[currentSong].audio);
   song.play();
+  playPauseBtn.setAttribute(
+    "aria-label",
+    `Playing ${playlist[currentSong].title}`
```

Comment:
```suggestion
    `Pause ${playlist[currentSong].title}`
```

Again, the button should say what it is going to do, not the state of the player.